internal/server/authn/method/github/server.go

```diff
@@ -165,26 +167,24 @@ func (s *Server) Callback(ctx context.Context, r *auth.CallbackRequest) (*auth.C
 	set(method.StorageMetadataName, githubUserResponse.Name)
 	set(method.StorageMetadataPicture, githubUserResponse.AvatarURL)
 
-	if len(s.config.Methods.Github.Method.AllowedOrganizations) != 0 {
-		userOrgs, err := getUserOrgs(ctx, token, apiURL)
-		if err != nil {
-			return nil, err
-		}
+	userOrgs, err := getUserOrgs(ctx, token, apiURL)
```

Comment:
note for another PR: we should consider caching these results in memory for a period of time as now we're doing these REST calls on every request if an org is using GitHub authn and the orgs/team membership do not change nearly that often